src/content/en/2020/resource-hints.md

```diff
@@ -158,36 +165,35 @@ As we create more and more automation with resource hints, be cautious when dyna
 
 {{ figure_markup(
   image="mobile-as-attribute-values-by-year.png",
-  caption="Mobile \"as\" attribute values by year.",
-  description="TODO",
+  caption="Mobile `as` attribute values by year.",
+  description="A bar chart comparing the rate of `as` attribute values on mobile pages from 2019 and 2020, broken down by `as` attribute value. The majority of `as` values are \"script\" with 81% usage in 2019 and 64% usage in 2020. \"script\" usage fell 17% year over year, while all other values increased in usage. \"not set\" increased 8%, \"font\" increased 5%, \"style\" increased 2%, the rest of the notable values are 1% or less for both years.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTYAbLxN40s6mNR1jo0XDe_V4siN8TAsx2mryMp5IQmlJ-9O9eJxYROz7Rw6ozyFP6hlIZHxxh95GqX/pubchart?oid=903180926&format=interactive",
   sheets_gid="1829901599",
   sql_file="as_attribute_by_year.sql"
 ) }}
 
-Compared to the trend in 2019, we've seen rapid growth in font and style usage with the `as` attribute. This is likely related to both developers increasing the priority of critical CSS and also combining `preload` fonts with `display:optional` to [improve](https://web.dev/optimize-cls/#web-fonts-causing-foutfoit) Cumulative Layout Shift ([CLS](https://web.dev/cls/)).
+Compared to the trend in [2019](../2019/resource-hints#the-as-attribute), we've seen rapid growth in font and style usage with the `as` attribute. This is likely related to developers increasing the priority of critical CSS and also combining `preload` fonts with `display:optional` to [improve](https://web.dev/optimize-cls/#web-fonts-causing-foutfoit) Cumulative Layout Shift ([CLS](https://web.dev/cls/)).
 
 Be mindful that omitting the `as` attribute, or having an invalid value will make it harder for the browser to determine the correct priority and in some cases, such as scripts, can even cause the resource to be fetched twice.
 
 ### The `crossorigin` attribute
 
 With `preload` and `preconnect` resources that have CORS enabled, such as fonts, it's important to include the `crossorigin` attribute, in order for the resource to be properly used. If the `crossorigin` attribute is absent, the request will follow the single-origin policy thereby making the use of preload useless.
 
-The latest trend indicates that by using `preload` with the `crossorigin` attribute we have 34% anonymous (or equivalent) cases, and 0.43% use-credentials cases. This has evolved in conjunction with the increase in font-preloading mentioned earlier.
+{# TODO(authors): Verify 34% is correct, and has support in the sheets results, and that the gid is correct. #}
```

Comment:
@Zizzamia

Reply:
The gid is correct `1185042785` but the data seems off 🤦🏻‍♂️, making a commit to fix it! Making a quick commit to fix it.